tor_archivist/core/blossom.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+class ConfigurationError(Exception):
+    pass
+
+class BlossomAPI(object):
+    def __init__(
+            self,
+            email: str=None,
+            password: str=None,
+            api_key: str=None,
+            api_base_url: str="http://api.grafeas.localhost:8000",
+            login_url: str="http://grafeas.localhost:8000/login/"
+    ):
+        if not email:
+            raise ConfigurationError("Need an email address!")
+        if not password:
+            raise ConfigurationError("Need a password!")
+        if not api_key:
+            raise ConfigurationError("Need an API key!")
+
+        self.email = email
+        self.password = password
+        self.base_url = api_base_url
+        self.login_url = login_url
+
+        self.s = requests.Session()
+        self.s.headers.update({'Authorization': f'Api-Key {api_key}'})
+
+    def _login(self):
```

Comment:
Just curious, why make this a private method (a python convention around the `_` prefix)? This seems stable enough to be public and you could take the email and password as args.

Reply:
@itsthejoker thoughts?

Reply:
I see nothing wrong with that :man_shrugging:, but let's address in a different PR.